ONSdigital/eq-author-app
# Ear 240 use new content picker

> BEFORE MAKING YOUR PR... There must be no linting errors and all tests must pass

### What is the context of this PR?

Swapped the old content picker to a newer version. This also involves deleting the old one. I've checked to see if it's used elsewhere but can't seem to find any other instances. Would appreciate a second pair of eyes though.

I think this PR has managed to swap over all the content pickers(famous last words). If any are noticed during review let me know, or about anything else.

**All modals should close when:**
1. clicking cross/cancel
2. clicking off the modal

**Should stay open when:**
1. Clicking the desktop outside of the browser window 

### How to review 
1. Create a questionnaire
2. Create two question pages both with a date answer
3. Enable earliest/latest date validation and select a previous answer
4. First question should resemble the first picture
5. Second question should resemble the second picture

| First Question  | Second Question |
| ------------- | ------------- |
| <img width="724" alt="Screenshot 2020-05-15 at 16 34 22" src="https://user-images.githubusercontent.com/22731314/82104045-31209300-96ca-11ea-98ca-e6ce6859afc7.png">|<img width="726" alt="Screenshot 2020-05-15 at 16 34 38" src="https://user-images.githubusercontent.com/22731314/82104057-3d0c5500-96ca-11ea-80b2-2f947e212629.png">|

1. Delete all metadata
2. Create a date range question
3. Try to enable earliest date validation with metadata
4. should get "no previous metadata to pick from"
5. Add a date variant of metadata back to the metadata
6.) Check metadata again and there should be an answer to choose from



### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
